src/OmniSharp.Host/WorkspaceInitializer.cs

```diff
@@ -27,11 +27,25 @@ public static void Initialize(IServiceProvider serviceProvider, CompositionHost
 
             var projectEventForwarder = compositionHost.GetExport<ProjectEventForwarder>();
             projectEventForwarder.Initialize();
-            var projectSystems = compositionHost.GetExports<IProjectSystem>();
 
             workspace.EditorConfigEnabled = options.CurrentValue.FormattingOptions.EnableEditorConfigSupport;
             options.OnChange(x => workspace.EditorConfigEnabled = x.FormattingOptions.EnableEditorConfigSupport);
 
+            logger.LogDebug("Starting with OmniSharp options: {options}", options.CurrentValue);
+            ProvideWorkspaceOptions(compositionHost, workspace, options, logger, omnisharpEnvironment);
```

Comment:
thanks, I think this is OK to change, however the `workspace.Initialized = true;` needs to be moved back down until after the project systems run their inits

Reply:
Oops, good catch.  Fixed